biz.aQute.bndlib/src/aQute/bnd/osgi/resource/CapReqBuilder.java

```diff
@@ -38,6 +41,10 @@
 
 public class CapReqBuilder {
 
+	private static final String			REQ_ALIAS_BUNDLE				= "bundle";
```

Comment:
Also `bundle` seems the wrong alias name. To me, a `bundle` requirement is `osgi.wiring.bundle` namespace. That is, `Require-Bundle`. You are doing an alias for `osgi.identity`, so I would think the better alias name is `bnd.identity`.